crates/forge/bin/cmd/verify/etherscan/mod.rs

```diff
@@ -321,7 +321,8 @@ impl EtherscanVerificationProvider {
         let mut verify_args =
             VerifyContract::new(args.address, contract_name, source, compiler_version)
                 .constructor_arguments(constructor_args)
-                .code_format(code_format);
+                .code_format(code_format)
+                .via_ir(args.via_ir);
```

Comment:
hmm, this option is completely retarded because in standardjson mode this info is already available in the input.

I feel like relying on the arg.via_ir is wrong, because if not provided this will set the option to false, even if viaIR is true in the StandardJsonCompilerInput

so I think we should only set this if true